docs/DeveloperGuide.md

```diff
@@ -505,38 +516,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case ends.
 
 
-**Use case: Importing data from .csv file**
+**Use case: UC 10. Importing data from [CSV](#glossary) file**
 
 **MSS**
 1. User has a data file with resident’s information, and wants to view it in RC4HDB.
-2. User uses the import command to import the CSV file.
-3. RC4HDB reads the file stored in the file path entered by the user.
-4. RC4HDB displays the name of the csv file after the file has been read.
+2. User imports the file.
+3. RC4HDB reads the file.
+4. RC4HDB displays the name of the [**CSV**](#glossary) file after the file has been read.
 5. RC4HDB displays all the information stored in the file.
 
     Use case ends.
 
-**Extension**
-* 1a. Information in the CSV file has not been stored in the proper format.
+**Extensions**
+* 2a. Information in the [**CSV**](#glossary) file has not been stored in the proper format.
```

Comment:
Should this not be 3a? Because the error crops out when reading the file right?

Reply:
I think 2a is still okay because ideally, you would want to detect the improper file format when it is imported.

Reply:
I agree with Naren, I think the import functionality should check for invalid file formats.

Reply:
Hmm i'm not too sure about this, we'll have to see how the import is implemented. IIRC, in the tutorial, if there was a change in the file (incorrect format), a blank list will be shown. So it depends on where the exception is thrown right? But not a major issue.

Reply:
Let's wait till I implement the `import` feature before we decide on this. I'll update the DG accordingly when it comes to it.